datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLModelMapper.java

```diff
@@ -242,4 +242,48 @@ private static void mapEditableProperties(MLModel entity, DataMap dataMap) {
     }
     entity.setEditableProperties(editableProperties);
   }
+
+  private static void mapVersionProperties(MLModel entity, DataMap dataMap) {
```

Comment:
Would expect a usage of this function above based on the aspect name, also + a test here